resources/troubleshooting-guides/README.md

```diff
@@ -0,0 +1,5 @@
+# Troubleshooting Guides
+
+This section has various articles which will help troubleshoot implementation issues in the SDK.
```

Comment:
Please reword so that it reads as follows:
"This section has various articles that will help you troubleshoot implementation issues in the SDK."